flutter_boilerplate/lib/module_mark/widget/mark_card.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+import 'package:pasco_shipping/module_mark/model/markModel.dart';
+import 'package:pasco_shipping/module_mark/response/mark_response.dart';
+import 'package:pasco_shipping/utils/styles/colors.dart';
+import 'package:pasco_shipping/utils/styles/text_style.dart';
+
+class MarkCard extends StatelessWidget {
+  final Mark model;
+  final String IncrementNumber;
```

Comment:
please rename this variable

Reply:
which one?? and why?

Reply:
IncrementNumber 
because it's not a class